onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -657,7 +657,6 @@ def create_multihead_attention_node(
             return None
 
         graph_input_names = set([node.name for node in self.model.graph().input])
-        graph_output_names = set([node.name for node in self.model.graph().output])
```

Comment:
Why is this deleted?

Reply:
This is not required since we are not checking if present_k or present_v is there in the output names, we are always appending it directly to output